app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/NoConnectionSimple.kt

```diff
@@ -0,0 +1,97 @@
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.aptoide_ui.theme.blueGradientEnd
+import cm.aptoide.pt.aptoide_ui.theme.blueGradientStart
```

Comment:
Lets not use the deprecated module

Reply:
error views have a different ticket. I just fixed the color to compile